### PR TITLE
Integrated resource bundle

### PR DESCRIPTION
**Pull request by michaelzangerle**
**Tasks:**
- [ ] gather feedback for my changes

**Informations:**

| Q | A |
| --- | --- |
| Tests pass? | none |
| Fixed tickets | none |
| BC Breaks | see UPGRADE.md |
| Doc | https://github.com/sulu-io/sulu-docs/pull/43 |

**Requires:**
- https://github.com/sulu-io/sulu/pull/1310

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -19,7 +25,7 @@ Service `sulu_security.user_repository` should be avoided. Use `sulu.repository.
 ### Snippets
 
 Snippet state has been removed and set default to published. Therefor all snippets has to be set to published by this
-running this command for each <locale>:
+running this command for each slocale>:
```
Comment (wachterjohannes) [inline UPGRADE.md]: here is a typo `<locale>`

File: UPGRADE.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Upgrade
 
+## Marked whereNot in ListBuilder as deprecated
```
Comment (wachterjohannes) [inline UPGRADE.md]: combine this two sections into one with a table: old - new

**Comment by wachterjohannes** [comment]
fixtures import is missing in upgrade

**Comment by wachterjohannes** [comment]
in filter list should be a backbutton to contact list

image

**Comment by wachterjohannes** [comment]
this command is missing in upgrade `app/console doctrine:schema:update --force`

**Comment by michaelzangerle** [comment]
We do not have back buttons on lists because they usually are the starting point! But I agree there should be some way to get back to the original list. Any other suggestions? Maybe we should ask Bernd!?

**Comment by wachterjohannes** [comment]
this will be open until all modules has integrated the filters!

**Pull request closed without merging**